target/bin/sedfile

```diff
@@ -33,14 +34,14 @@ fi
 FILE=${*: -1}
 
 OLD=$(${HASHTOOL} "${FILE}")
-sed "$@"
+sed "${@}"
 NEW=$(${HASHTOOL} "${FILE}")
 
 # fail if file was not modified
 if [[ ${OLD} == "${NEW}" ]] && [[ ${SKIP_ERROR} -eq 0 ]]
 then
-  echo "Error: sed $*"
+  echo "No difference after call to 'sed' in 'sedfile' (sed ${*})" >&2
```

Comment:
Same here. Why was `Error:` removed? If this line is printed, a change was expected, but did not take place. Therefore an error occured.

Reply:
Done in [`ed512d0`](https://github.com/docker-mailserver/docker-mailserver/pull/2500/commits/ed512d04830330fda13343f49f8d7621249ac424)